### PR TITLE
Fix WindowsClipboard.SetText being broken

### LINKED ISSUE #1572: WindowsClipboard.SetText is broken
`
[runtime:important] System.EntryPointNotFoundException: Unable to find an entry point named 'CopyMemory' in DLL 'kernel32.dll'.
   at osu.Framework.Platform.Windows.WindowsClipboard.CopyMemory(IntPtr dest, IntPtr src, UInt32 count)
   at osu.Framework.Platform.Windows.WindowsClipboard.SetText(String selectedText) in C:\osu\osu-framework\osu.Framework\Platform\Windows\WindowsClipboard.cs:line 124
`
This causes a crash on Net Core builds when trying to copy a text.
According to [msdn](https://msdn.microsoft.com/en-us/library/windows/desktop/aa366535(v=vs.85).aspx),
> This function is defined as the RtlCopyMemory function. Its implementation is provided inline. For more information, see WinBase.h and WinNT.h.

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/58957225-windowsclipboard-settext-is-broken?utm_campaign=plugin&utm_content=tracker%2F44024461&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F44024461&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by ekrctb**
Fixes #1572.

### CODE REVIEW

**Comment by UselessToucan** [comment]
Did you check this on both platforms? I mean netcore and net471 on Windows.

File: osu.Framework/Platform/Windows/WindowsClipboard.cs [hunk 1]
```diff
@@ -121,7 +121,7 @@ public override void SetText(string selectedText)
 
                     try
                     {
-                        CopyMemory(target, source, bytes);
+                        RtlCopyMemory(target, source, (UIntPtr)bytes);
```
Comment (smoogipoo) [inline osu.Framework/Platform/Windows/WindowsClipboard.cs]: Would rather use `System.Buffer.MemoryCopy` and avoid the pinvoke.

**Comment by ekrctb** [comment]
I have tested on both net471 and netcore.

**Comment by peppy** [comment]
Seems to work correctly on both runtimes 👍

**Pull request merged**